docs/source/index.rst

```diff
@@ -9,15 +13,32 @@ On a single machine dask increases the scale of comfortable data from
 *fits-in-memory* to *fits-on-disk* by intelligently streaming data from disk
 and by leveraging all the cores of a modern CPU.
 
-Users interact with dask either by making graphs directly or through the *dask
-collections* which provide larger-than-memory counterparts to existing popular
-libraries:
+Dask use cases
+==============
+
+* **Scientific computing libraries:** Projects like `xray`_ and `scikit-image`_ 
+  use dask.array for fast interactive analysis of large numeric datasets.
+* **Large unstructured or semi-structured data:** Dask.bag excels at 
+  processing large JSON blobs and log files.
+* **Scaling pandas-like data analysis to large datasets:** Dask.dataframe 
+  implements a part of the pandas interface, and can process data files larger than 
+  available memory.
+* **Algorithm development:** Working directly with dask graphs or dask.imperative 
+  is an excellent way to implement and test new algorithms in fields such as linear 
+  algebra, optimization, and machine learning.
```

Comment:
This section feels a lot better than it did before.  I still wonder if it's a bit further ahead in the docs than it ought to be.  At this stage readers don't know what `dask.bag` or `dask.array` are, right?  While I like the idea of referring to other projects it seems odd to point out two particular projects really early on unless they're really critical to dask (which, in this case, I don't think that they are).

In general this section feels like it collects an odd collection of details before general trends have been established.  We're at line 16 of the docs, we should be focusing on general use cases, not specific ones.  Or, if we want to have specific use cases then we should prompt them first with a general trend, then a specific example.

I think that what happened here is that someone asked me "what are the use cases of dask" and I said "It really depends on the population.  There are a lot of very varied use cases.  For example here are a bunch of scattered use cases."  Then someone blindly copied my e-mail into the docs under a use case section.  Now you're trying to clean up that original misguided copy-paste.  We should maybe rethink this section from the ground up.